it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -636,7 +690,8 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
           stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoNino).setNewScenarioState("auditing")
           stubRemoveInsights(testJourneyId)(NO_CONTENT)
           stubRetrieveSaPostcode(testJourneyId)(OK, testPostcode)
-          stubRetrieveOverseasTaxIdentifiers(testJourneyId)(OK, testOverseasTaxIdentifiersJson)
+          stubRetrieveOverseasTaxIdentifier(testJourneyId)(OK, testOverseasTaxIdentifier)
```

Comment:
I'm not certain you need to stub the overseas tax identifier methods here.

Reply:
done